src/structures/VoiceState.js

```diff
@@ -138,6 +139,34 @@ class VoiceState extends Base {
     return this.member ? this.member.edit({ deaf }, reason) : Promise.reject(new Error('VOICE_STATE_UNCACHED_MEMBER'));
   }
 
+  /**
+   * Self-mutes/unmutes the bot for this voice state.
+   * @param {boolean} mute Whether or not the bot should be self-muted
+   * @returns {Promise<boolean>}
```

Comment:
You might be able to infer the meaning of the return value, but a doc-string would certainly not hurt.